plugins/obs-remote

```diff
@@ -0,0 +1 @@
+../../obs-remote/
```

Comment:
This symbolic link points to a folder that is outside the project root.

Reply:
Lol this is what happens when i build in a dirty working dir. Thought I caught this, thanks.